Datadog/Datadog.xcodeproj/xcshareddata/xcschemes/Example.xcscheme

```diff
@@ -99,7 +99,7 @@
          <EnvironmentVariable
             key = "DD_TEST_SCENARIO_IDENTIFIER"
             value = "RUMResourcesScenario"
-            isEnabled = "NO">
+            isEnabled = "YES">
```

Comment:
This should not be committed as it's modifying the shared scheme for everyone 😉.